lib/api/core/models/repositories/roleRepository.js

```diff
@@ -185,13 +201,13 @@ class RoleRepository extends Repository {
    */
   deleteRole (role, options = {}) {
     if (['admin', 'default', 'anonymous'].indexOf(role._id) > -1) {
-      return Promise.reject(new BadRequestError(role._id + ' is one of the basic roles of Kuzzle, you cannot delete it, but you can edit it.'));
+      return Bluebird.reject(new BadRequestError(role._id + ' is one of the basic roles of Kuzzle, you cannot delete it, but you can edit it.'));
     }
 
     return this.kuzzle.repositories.profile.searchProfiles([role._id], {from: 0, size: 1})
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 90 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)